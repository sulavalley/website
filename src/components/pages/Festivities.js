import React from 'react';

const Festivities = () => {
  return (
    <div>
      <h1>Estas en Festividades</h1>
    </div>
  );
};

export default Festivities;
