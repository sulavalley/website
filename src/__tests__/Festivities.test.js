import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Festivities from './../components/pages/Festivities';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Festivities />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Festivities />);
});

test('renders correctly', () => {
  const tree = renderer.create(<Festivities />).toJSON();
  expect(tree).toMatchSnapshot();
});
