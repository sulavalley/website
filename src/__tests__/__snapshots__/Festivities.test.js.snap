// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Estas en Festividades
  </h1>
</div>
`;
